### DISCUSSION TITLE
How can I disable blending effect in iOS devices?

### DISCUSSION THREAD

**Discussion by DuhDamn** [google/model-viewer#2243] [Q&A]
### Description

I have been using Model Viewer to add WebAR onto my website, On android the models looks good and functions properly, but for iOS, the model is blending into atmosphere which is not what I need, How can I disable it using Model Viewer api? I couldn't find any references, apart from this one github issue https://github.com/google/model-viewer/issues/1733#issuecomment-749136234 , but when I use it, I get an error saying "Object requires newer version of iOS".

#### Live Demo
Image of the Issue I'm haing: https://i.stack.imgur.com/PFizj.jpg
Image of the Error: https://i.stack.imgur.com/ZvkLD.jpg

### Version

- model-viewer: vX.X.X

### Browser Affected
Safari

### OS

iOS

**Comment by elalish** [comment]
This is a QuickLook bug and the intent solution you refer to is for SceneViewer. I'm not aware of Apple allowing this kind of customization in QuickLook, but if you find a way in their docs, I'll be happy to add it. I'm guessing you'll get better results with a new iPhone 12 with LIDAR.
